.github/workflows/tests_cuda.yml

```diff
@@ -0,0 +1,31 @@
+on: [pull_request]
```

Comment:
Shouldn't we run the gpu test also on push?

Reply:
sure. I'll fix